cpp/include/cudf/io/parquet.hpp

```diff
@@ -567,9 +585,25 @@ class parquet_writer_options {
   {
     CUDF_EXPECTS(
       size_rows >= 5000,
-      "The maximum row group size cannot be smaller than the page size, which is 5000 rows.");
+      "The maximum row group size cannot be smaller than the fragment size, which is 5000 rows.");
     _row_group_size_rows = size_rows;
   }
+
+  /**
+   * @brief Sets the maximum uncompressed page size, in bytes.
+   */
+  void set_max_page_size_bytes(size_t size_bytes) { _max_page_size_bytes = size_bytes; }
```

Comment:
I think this should have a similar logic to `set_max_page_size_rows` and error below a hardcoded threshold.

Reply:
@vuule What would you suggest as a lower bound?  Parquet itself is silent on this, is there a practical minimum size dictated by the compressors?  I would need to be able to go at least as low as 64K.

Reply:
4KB maybe? I'm pretty much making this up based on your requirement and assumed 512 limit for rows.

Reply:
4KB works for me.  Done.